index.html

```diff
@@ -23,60 +23,85 @@
      </div>
   </header>  
   <main id="section-main">
-        <h1>AMBROSE <br> ISIGBA</h1>
-        <p>Hello! I am a passionate software developer! my goal
+     <div id ="section-main-text">
+         <h1>AMBROSE ISIGBA</h1>
+         <p>Hello! I am a passionate software developer! my goal
            is to help you build a quality product(s), features 
            and/or website with international standard. You are 
-           highly invited to take a tour of my works.If you fall in 
-            love with  my work gallery, and you are planning for one or 
-           you need a code mentor. AMBROSE's CODELAB get you covered</p>
-        <button type="button">Start colaboration</button>
-        <div id="second-section-bg-img"></div>
-  </main>
-  <section id="project-sections">
-     <h2>Projects</h2>
+           highly invited to take a tour of my works.</p>
+          <button type="button">Start colaboration <img id="btn-imgA" src="./images/image_dots.png" alt=""></button>
+      
+        </div>
+        <div id="section-main-img2"> 
+      <div id="second-section-bg-img"><img src="./images/Shape1b.png" alt=""></div>
+      <div><img  src="./images/image-geometry1.png" alt=""></div>
+    </div>
+    </main>
+   <section id="project-sections">
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the project section in the desktop view has the proper margins, this will make your design closer to the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_052709](https://user-images.githubusercontent.com/25227022/173172431-f7096643-3519-4123-b79f-e995ecd3e47c.png)|![Screenshot_20220611_052940](https://user-images.githubusercontent.com/25227022/173172492-4b4e8284-2062-4ff1-8e55-e67ce3e786c6.png)|
|![Screenshot_20220611_052755](https://user-images.githubusercontent.com/25227022/173172432-1b471d39-b7c5-4dd1-95bb-c7547edc2d9b.png)|![Screenshot_20220611_052957](https://user-images.githubusercontent.com/25227022/173172498-adbd14c5-b746-45b6-a28f-35edffdf82f2.png)|